analyzeTest.ts

```diff
@@ -0,0 +1,55 @@
+import * as fs from 'fs';
+
+const pluginName = process.env.PLUGIN;
+
+const getStatsFilePath = () => `./plugins/${pluginName}/dist/stats.json`;
+
+const toMB = (value: number) => value / 1024 ** 2;
+
+const stringifyMB = (value: ReturnType<typeof toMB>): string =>
+  `${value.toFixed(2)} MB`;
+
+const getParsedStatFile = () => {
+  const filePath = getStatsFilePath();
+  const file = fs.readFileSync(filePath, 'utf-8');
+  const statsData = JSON.parse(file);
+  return statsData;
+};
+
+type BundleDataMap = Record<string, number>;
+
+// [Valid Bundles, Violating Bunldes]
+type GetBundleInformation = () => [BundleDataMap, BundleDataMap];
+
+const getBundleInformation: GetBundleInformation = () => {
+  const statsData = getParsedStatFile();
+  const validAssets = {};
+  const violatingAssets = {};
+
+  statsData.assets.forEach((asset) => {
+    const assetSize = toMB(asset.size);
+    const readableSize = stringifyMB(assetSize);
+    if (assetSize > 3) {
```

Comment:
Just a naming suggestion so the role of the number 3 is more intentional.
```suggestion
    const maxAllowedSize = 3;
    if (assetSize > maxAllowedSize) {
```